examples/vertx-opentracing/pom.xml

```diff
@@ -139,4 +138,20 @@
     </plugins>
   </build>
 
+  <repositories>
```

Comment:
Otherwise it failed to resolve the opentracing client, and the trace publisher rest client artifacts.